stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/HomeViewModel.kt

```diff
@@ -55,6 +56,9 @@ class HomeViewModel @Inject constructor(
     val facility: StateFlow<OrganisationUnit?>
         get() = _facility
 
+    private val _scanText = MutableStateFlow("")
+    val scanText = _scanText.asStateFlow()
```

Comment:
don't you need here the `get() = ...`?

Reply:
asStateFlow() implicitly implements get() method, but to make uniform I have added. What do you think?